server/tserver/src/main/java/org/apache/accumulo/tserver/compactions/CompactionManager.java

```diff
@@ -298,7 +298,9 @@ private synchronized void checkForConfigChanges(boolean force) {
 
   public void start() {
     log.debug("Started compaction manager");
-    Threads.createThread("Compaction Manager", () -> mainLoop()).start();
+    // TODO KEVIN RATHBUN This is a critical thread for the TabletServer to run properly and is
+    // only called once.
+    Threads.createCriticalThread("Compaction Manager", () -> mainLoop()).start();
```

Comment:
This is a critical thread for the TabletServer to run properly and is only called once.